src/pages/Account/AccountPayment.tsx

```diff
@@ -61,24 +61,26 @@ const AccountPayment = ({ creditBalance }: Props) => {
 
             <Row>
               <Col xs="12">
-                <div onClick={() => handleModal(ModalType.ADD_NEW_CARD)} className="w-auto d-inline-block align-items-center">
-                  <i className="fas fa-plus-circle" />
-                  <h6 className="ml-2 mb-0 d-inline-block">Add New Card</h6>
-                </div>
+                <a onClick={() => handleModalAction(ModalType.ADD_NEW_CARD)}>
```

Comment:
doesn't reactstrap button handle icon inside the link for you?

Reply:
hmm are there any examples of this?

Reply:
i added the facebook icon next to the text. The padding can be improved but it does show an icon button.
![screen shot 2019-02-20 at 3 08 12 pm](https://user-images.githubusercontent.com/7976/53131337-62d79800-3521-11e9-9cb9-9bfe6c33c594.png)

Reply:
thanks. updated